source/install.rst

```diff
@@ -1 +1,297 @@
-1. Setting up your environment
+############
+Installation
+############
+
+Before we start, we need to prepare our environment. Below are some instructions on installing the latest version of Python. Here we install version 3.6.5.
+
+Windows
+=======
+
+You can download Python for Windows directly from `python.org`_. Look for version 3 and above.
+After downloading the file  ``*.msi``, open it and follow the instructions.
+It is important to remember the path of installation, i.e. the directory, as we will need this information during the :ref:`installation of tools <tools>`.
+
+.. note::
+    Remember to check "Add Python 3.6 to PATH" when running the installer . This way you will be able to run Python form
+    console by just writing ``python``.
+
+
+Linux (Ubuntu, Fedora, etc.) or Mac
+===================================
+
+In order to check our version of Python, enter the following in the command line:
+
+.. code-block:: sh
+
+    $ python --version
+    Python 3.6.5
+
+If the ``python`` command is not available or the wrong version appears try using ``python3`` command:
```

Comment:
Something missing here? Or maybe it should just be "." instead of ":" ;) But then I would add a sentence below, like "If python is not installed, follow these instructions."